robot/packages/cloud_package_manager.go

```diff
@@ -202,6 +214,19 @@ func (m *cloudManager) Sync(ctx context.Context, packages []config.PackageConfig
 	return outErr
 }
 
+// modulesForPackage returns module(s) whose ExePath is in the package's directory.
+// TODO: This only works if you call it after placeholder replacement. Find a cleaner way to express this link.
```

Comment:
Yeah -- this lack-of-link is one of the primary reasons why I've been such a big advocate of adding packages and modules the graph